.gitignore

```diff
@@ -1,5 +1,9 @@
 **/.DS_Store
 .circleci/generated_config.yml
+scripts/cci-job.data.json
+scripts/cci-test.data.json
+artifacts/cci-workflow-results.json
+artifacts/cci-workflow-results.html
```

Comment:
these files are intermediates (we only use them temporarily, and discard after processing)